uptime-service-backend/templates/deployment.yaml

```diff
@@ -7,29 +7,29 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "common.names.fullname" . }}
+  name: {{ include "uptime-service-backend.fullname" . }}
   labels:
-    {{- include "common.labels.standard" . | nindent 4 }}
+    {{- include "uptime-service-backend.labels" . | nindent 4 }}
 spec:
   {{- if not .Values.autoscaling.enabled }}
-  replicas: {{ .Values.replicas }}
+  replicas: {{ .Values.replicaCount }}
   {{- end }}
   selector:
     matchLabels:
-      app: {{ .Release.Name }}
+      {{- include "uptime-service-backend.selectorLabels" . | nindent 6 }}
   template:
     metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
       labels:
-        app: {{ .Release.Name }}
+        {{- include "uptime-service-backend.labels" . | nindent 8 }}
+	{{- with .Values.podLabels }}
```

Comment:
**Nitpick:** I think there is a mix between tab and space here.
You could eventually add a plugin to you text editor to fix this on the save command

Reply:
I copied those lines from the `helm create` command, I am confused on why that got there.

I will look into my editor so it highlights these before I commit. Thanks for the tip!